.github/workflows/test_destruction.yml

```diff
@@ -14,11 +14,14 @@ jobs:
       licenceplate: ${{ secrets.LICENCEPLATE}}
       tfc_team_token: ${{ secrets.TFC_TEAM_TOKEN }}
       dev_oidc_idir_idp_client_secret: "${{ secrets.DEV_OIDC_IDIR_IDP_CLIENT_SECRET }}"
-      test_oidc_bceid_business_idp_client_secret: "${{ secrets.TEST_OIDC_IDIR_IDP_CLIENT_SECRET }}"
```

Comment:
The values here don't actually get used because there is no deployment going on (it's a DESTROY). I fixed them because I figured someone would see that they were different from the other ones and get confused.